### ISSUE TITLE
Parse variable references in `terminal.integrated.env.osx`

### ISSUE THREAD

**Issue by JakobGM** [innoverio/vscode-dbt-power-user#97]
Hi, there! 👋 

The [current recommendation](https://github.com/innoverio/vscode-dbt-power-user/issues/91) for users of this plugin that wish to forward environment variables to the execution environment of `dbt` is to set `terminal.integrated.env.{osx,linux,windows}`. The problem is that the [current implementation](https://github.com/innoverio/vscode-dbt-power-user/blob/master/src/dbt_client/index.ts#L177-L184) does not support [variable references](https://code.visualstudio.com/docs/editor/variables-reference). With other words, this does not work:

```json
{
    "terminal.integrated.env.osx": {
        "DBT_PROFILES_DIR": "${workspaceFolder}",
        "SOME_VARIABLE_USED_BY_DBT": "${env:SOME_VARIABLE_USED_BY_DBT}"
    }
}
```

This works for regular integrated terminals, so it is a divergence from the normal behavior of VS code.

An even better solution to this problem would be if `terminal.integrated.inheritEnv` is set to `true`, then `dbt` is executed in a terminal with the exact same environment variables as the host. Would that be possible? I think it might even make sense as the default behavior.

Thanks in advance, whatever you decide!

**Comment by Lef-F** [comment]
We're having the same problem. This is such an awesome extension but we cannot use it out-of-the-box because we maintain the `profiles.yml` file inside our `dbt` project. It would be amazing to support a simple configuration in e.g. `settings.json`.

**Comment by mdesmet** [comment]
This should be easy to implement by looking up how this is done in the vscode Python extension.

**Comment by nc-pebl** [comment]
We're having the same issue as @Lef-F when using poetry. Is there any plan to support this in the near future?

**Comment by mdesmet** [comment]
That can easily be solved today by setting `export DBT_PROFILES_DIR=.` or by setting `dbt.profilesDirOverride` to `.` in vscode settings.

**Comment by mdesmet** [comment]
Note also all host environment variables are passed down to the process invocation. it is just the syntax of variable references in vscode settings that is currently not supported.

**Comment by mdesmet** [comment]
Fixed in 0.8.9. Let me know if it works
